tests/foreman/api/test_repository.py

```diff
@@ -20,7 +20,8 @@
 from fauxfactory import gen_string
 from nailgun import client, entities
 from nailgun.entity_mixins import TaskFailedError
-from requests.exceptions import HTTPError, SSLError
+from OpenSSL.SSL import Error
```

Comment:
prefere (and do nothing with the test)
```python
from OpenSSL.SSL import Error as SSLError
```

Reply:
@ldjebran, to keep exception name isolated/uniq I used "Error" here.
if we use the alias then again that is a same name exception which is already provided with 'requests' module.